setup.py

```diff
@@ -5,7 +5,7 @@
 from setuptools import setup
 
 GARAGE_GH_TOKEN = os.environ.get('GARAGE_GH_TOKEN') or 'git'
-TF_VERSION = '<1.16,>=1.15.0'
+TF_VERSION = '==2.1'
```

Comment:
does this actually work? i think it requires a lot of fixes.

if it does, just unpin TF entirely.